Error while trying to install from cargo

**Describe the bug**
Error while trying to install from [cargo](https://github.com/denisidoro/navi/blob/master/docs/installation.md#using-cargo) or [install script](https://github.com/denisidoro/navi/blob/master/docs/installation.md#using-install-script). Both give the same error.

**To Reproduce**
Just run the required installation code.

**Expected behavior**
To install it.

**Screenshots**
```
$ cargo install navi
    Updating crates.io index
  Installing navi v2.17.0
   Compiling libc v0.2.100
   Compiling cfg-if v1.0.0
   Compiling proc-macro2 v1.0.28
   Compiling unicode-xid v0.2.2
   Compiling syn v1.0.75
   Compiling version_check v0.9.3
   Compiling memchr v2.4.1
   Compiling log v0.4.14
   Compiling autocfg v1.0.1
   Compiling ppv-lite86 v0.2.10
   Compiling signal-hook v0.3.9
   Compiling serde_derive v1.0.128
   Compiling scopeguard v1.1.0
   Compiling serde v1.0.128
   Compiling smallvec v1.6.1
   Compiling unicode-segmentation v1.8.0
   Compiling hashbrown v0.11.2
   Compiling bitflags v1.3.2
   Compiling utf8parse v0.1.1
   Compiling regex-syntax v0.6.25
   Compiling lazy_static v1.4.0
   Compiling anyhow v1.0.43
   Compiling either v1.6.1
   Compiling linked-hash-map v0.5.4
   Compiling remove_dir_all v0.5.3
   Compiling unicode-width v0.1.8
   Compiling termcolor v1.1.2
   Compiling same-file v1.0.6
   Compiling dtoa v0.4.8
   Compiling os_str_bytes v3.1.0
   Compiling vec_map v0.8.2
   Compiling strsim v0.10.0
   Compiling instant v0.1.10
   Compiling lock_api v0.4.4
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling vte v0.3.3
   Compiling indexmap v1.7.0
   Compiling textwrap v0.14.2
   Compiling yaml-rust v0.4.5
   Compiling walkdir v2.3.2
   Compiling heck v0.3.3
   Compiling strip-ansi-escapes v0.1.0
   Compiling aho-corasick v0.7.18
   Compiling quote v1.0.9
   Compiling getrandom v0.2.3
   Compiling signal-hook-registry v1.4.0
   Compiling mio v0.7.13
   Compiling parking_lot_core v0.8.3
   Compiling atty v0.2.14
   Compiling which v4.2.2
   Compiling dirs-sys-next v0.1.2
   Compiling remove_dir_all v0.7.0
   Compiling rand_core v0.6.3
   Compiling parking_lot v0.11.1
   Compiling regex v1.5.4
   Compiling directories-next v2.0.0
   Compiling rand_chacha v0.3.1
   Compiling signal-hook-mio v0.2.1
   Compiling shellwords v1.1.0
   Compiling rand v0.8.4
   Compiling crossterm v0.20.0
   Compiling tempfile v3.2.0
   Compiling edit v0.1.3
   Compiling thiserror-impl v1.0.26
   Compiling clap_derive v3.0.0-beta.4
   Compiling thiserror v1.0.26
   Compiling clap v3.0.0-beta.4
error[E0658]: arbitrary expressions in key-value attributes are unstable
 --> /home/gca/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-3.0.0-beta.4/src/lib.rs:8:10
  |
8 | #![doc = include_str!("../README.md")]
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: see issue #78835 <https://github.com/rust-lang/rust/issues/78835> for more information

error[E0658]: use of unstable library feature 'osstring_ascii'
   --> /home/gca/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-3.0.0-beta.4/src/parse/matches/matched_arg.rs:130:19
    |
130 |                 v.eq_ignore_ascii_case(val)
    |                   ^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #70516 <https://github.com/rust-lang/rust/issues/70516> for more information

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
error: could not compile `clap`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `navi v2.17.0`, intermediate artifacts can be found at `/tmp/cargo-installph3cZ6`

Caused by:
  build failed
```

**Versions:**
 - OS: Ubuntu 21.04
 - Shell Version: GNU bash, version 5.1.4(1)-release (x86_64-pc-linux-gnu)
 - Cargo Version: 1.51.0

**Additional context**
Installed cargo via apt-get. Successfully installed other stuff via cargo.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,7 +16,7 @@ travis-ci = { repository = "denisidoro/navi", branch = "master" }
 
 [dependencies]
 regex = { version = "1.5.4", default-features = false, features = ["std", "unicode-perl"] }
-clap = "3.0.0-beta.4"
+clap = "3.0.0-beta.2"
 crossterm = "0.20.0"
 lazy_static = "1.4.0"
 directories-next = "2.0.0"

